presto-parser/src/main/java/io/prestosql/sql/tree/ShowCreate.java

```diff
@@ -28,7 +28,8 @@
     public enum Type
     {
         TABLE,
-        VIEW
+        VIEW,
+        SCHEMA
```

Comment:
add:
```
SCHEMA,
/**/;
```
That way you can add new entries with changing only single line.

Reply:
Done.